### PR TITLE
LCH-2535 | Provide function to add entities to Interest Lists

### PR DESCRIPTION
**Pull request by Ateee89**

### CODE REVIEW

File: src/ContentHubClient.php [hunk 1]
```diff
@@ -622,6 +622,24 @@ public function updateWebhook($uuid, $url)
       return $this->put("/settings/webhooks/{$uuid}", $options) ;
     }
 
+  /**
```
Comment (jabxjab) [inline src/ContentHubClient.php]: @Ateee89 Please fix indentation of the phpdoc block.

File: src/ContentHubClient.php [hunk 2]
```diff
@@ -622,6 +622,24 @@ public function updateWebhook($uuid, $url)
...
+   * Add entities to Intrest List.
+   *
+   * @param $webhookUuid
```
Comment (jabxjab) [inline src/ContentHubClient.php]: @Ateee89 Missing types

File: src/ContentHubClient.php [hunk 3]
```diff
@@ -622,6 +622,24 @@ public function updateWebhook($uuid, $url)
...
+   *   The UUID of the webhook
+   * @param $uuids
+   *   Entity UUIDs to add to Interest List
+   *
+   * @return \Psr\Http\Message\ResponseInterface
```
Comment (jabxjab) [inline src/ContentHubClient.php]: @Ateee89 Missing `return` description

**Review by jabxjab — APPROVED** [review]
Please fix minor code style issues

File: src/ContentHubClient.php [hunk 4]
```diff
@@ -622,6 +622,25 @@ public function updateWebhook($uuid, $url)
...
+     *   The UUID of the webhook
+     * @param array $uuids
+     *   Entity UUIDs to add to Interest List
+     *
+     * @return \Psr\Http\Message\ResponseInterface
+     *   The response.
+     *
+     * @throws \GuzzleHttp\Exception\RequestException
+     */
+    public function addEntitiesToInterestList($webhookUuid, $uuids)
```
Comment (abarriosr) [inline src/ContentHubClient.php]: Use small letters to variable name like `$webhook_uuid`

**Pull request merged**